Trading/Exchange/binance/binance_exchange.py

```diff
@@ -29,6 +29,15 @@ class Binance(exchanges.SpotCCXTExchange):
 
     BINANCE_MARK_PRICE = "markPrice"
 
+    BINANCE_CCXT_OPTIONS = {
+        "timeDifference": 0,  # the difference between system clock and Binance clock
+        "adjustForTimeDifference": False,  # controls the adjustment logic upon instantiation
```

Comment:
if this fixes the invalidNonce error, that would be amazing 🎉

Reply:
I hope so!